workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestList/utils/getMDFListColumns.tsx

```diff
@@ -29,6 +30,23 @@ export default function getMDFListColumns(
 	mutated?: KeyedMutator<LiferayItems<MDFRequestDTO[]>>,
 	isChannel?: boolean
 ): TableColumn<MDFRequestListItem>[] | undefined {
+	const getCustomColumn = (
+		row: MDFRequestListItem,
+		column1: string,
+		column2: string
+	) => {
+		return (
+			<div>
+				<p className="border-0 font-weight-normal mb-0 text-truncate text-truncate-inline">
+					{row[column1 as MDFColumnKey]}
+				</p>
+				<p className="mb-0 mt-0 text-neutral-7 text-paragraph-sm">
+					{row[column2 as MDFColumnKey]}
+				</p>
+			</div>
+		);
+	};
```

Comment:
Think about simplicity, I think you can pass this piece of code instead have this function